src/SwiftUIGlue/Glue.swift

```diff
@@ -198,3 +198,10 @@ public func GetRGBColorSpacesRGBLinear(dest : UnsafeMutableRawPointer)
     let sRGBLinear = Color.RGBColorSpace.sRGBLinear
     dest.initializeMemory(as: type(of: sRGBLinear), repeating: sRGBLinear, count: 1)
 }
+
+@_silgen_name("swiftui_View_modifier")
+public func SetViewModifier<TView: View, TViewModifier: ViewModifier>(dest : UnsafeMutableRawPointer, view : TView, value : TViewModifier)
```

Comment:
We know the return type of `view.modifier`-- it's `SwiftUI.ModifiedContent<TView, TViewModifier>`--so this should use the generic `UnsafeMutablePointer`:
```suggestion
public func SetViewModifier<TView: View, TViewModifier: ViewModifier>(dest : UnsafeMutablePointer<ModifiedContent<TView, TViewModifier>>, view : TView, value : TViewModifier)
```